source/extensions/filters/http/ext_proc/on_processing_response.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include "envoy/config/typed_config.h"
+#include "envoy/server/factory_context.h"
+#include "envoy/service/ext_proc/v3/external_processor.pb.h"
+#include "envoy/stream_info/stream_info.h"
+
+#include "source/common/stats/symbol_table.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+class OnProcessingResponse {
```

Comment:
would be better to add more comments on top of the class to explain more for the interface usage.